projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -191,6 +199,7 @@ protected function inject_javascript_options() {
 			'hasOverlayWidgets'     => count( $overlay_widget_ids ) > 0,
 			'widgets'               => array_values( $widgets ),
 			'widgetsOutsideOverlay' => array_values( $widgets_outside_overlay ),
+			'hasOtherWidgets'       => $has_other_widgets,
```

Comment:
When we say "other widgets", what counts as "normal" and what counts as "other"?

I wonder if there's a more descriptive name we can use (or if not, we could add some comments to explain what the "other widgets" includes).

Reply:
You are right. The name is a bit ambiguous. I'll give it a better name.